Source/Plugins/OrientationAnalysis/Documentation/OrientationAnalysisFilters/ReadCtfData.md

```diff
@@ -19,37 +19,33 @@ If the data has come from a HKL acquisition system and the settings of the acqui
 
 The user also may want to assign un-indexed pixels to be ignored by flagging them as "bad". The [Threshold Objects](@ref multithresholdobjects) **Filter** can be used to define this _mask_ by thresholding on values such as _Error_ = 0.
 
-
 ### Radians and Degrees ###
 
 2D .ctf files have their angles in **degrees**. Please be sure to [convert](@ref changeanglerepresentation) the Euler angles from degrees to radians before running any other **Filter**.
```

Comment:
Should add a line here about the option for the filter to do the conversion for the user instead of needing to use the  convert angles filter.

Reply:
Done.